armbian/base/scripts/bbb-config.sh

```diff
@@ -0,0 +1,210 @@
+#!/bin/bash
+set -e
+
+# BitBox Base: system configuration utility
+#
+
+SYSCONFIG_PATH="/opt/shift/sysconfig/"
+mkdir -p "$SYSCONFIG_PATH"
+
+function usage() {
+  echo "BitBox Base: system configuration utility
+usage: bbb-config [--version] [--help]
+                  <command> [<args>]
+
+possible commands:
+  enable    <dashboard_hdmi|dashboard_web|wifi|autosetup_ssd>
+
+  disable   any 'enable' argument
+
+  set       <bitcoin_network|hostname|root_pw|wifi_ssid|wifi_pw>
+            bitcoin_network     <mainnet|testnet>
+            other arguments     string
+
+  get       any 'enable' or 'set' argument, or <all>
+"
+}
+
+if [[ "$#" -eq 0 ]] || [[ "$1" == "-h" ]] || [[ "$1" == "--help" ]]; then
+  usage
+  exit 0
+elif [[ "$1" == "-v" ]] || [[ "$1" == "--version" ]]; then
+  echo "bbb-config version 0.1"
+  exit 0
+fi
+
+if [[ ${UID} -ne 0 ]]; then
+  echo "${0}: needs to be run as superuser." >&2
+  exit 1
+fi
+
+COMMAND="${1}"
+SETTING="${2^^}"
+
+case "${COMMAND}" in
+    enable|disable)
+        if [[ "${COMMAND}" == "enable" ]]; then
+            ENABLE=1
+        else
+            ENABLE=0
+        fi
+
+        case "${SETTING}" in
+            DASHBOARD_HDMI)
+                # enable / disable auto-login for user "hdmi", start / kill xserver
+                if [[ $ENABLE -eq 1 ]]; then
```

Comment:
If we're using curly braces (`${VAR}`), I think we should do it consistently, i.e `${ENABLE}`.

(And elsewhere.)